src/status_im2/contexts/quo_preview/settings/accounts.cljs

```diff
@@ -0,0 +1,48 @@
+(ns status-im2.contexts.quo-preview.settings.accounts
+  (:require [clojure.string :as string]
+            [quo2.components.settings.accounts.view :as quo2]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [reagent.core :as r]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label   "Custom color"
+    :key     :custom-color
+    :type    :select
+    :options (mapv (fn [[k _]]
+                     {:key k, :value (string/capitalize (name k))})
+                   colors/customization)}
+   {:label "Account name"
+    :key   :account-name
+    :type  :text}
+   {:label "Account address"
+    :key   :account-address
+    :type  :text}])
+
+(defn cool-preview []
+  (let [state (r/atom {:custom-color    :blue
+                       :account-name    "Booze for Dubai"
+                       :account-address "0x21a ... 49e"
+                       ;; Fixed parameters:
```

Comment:
This comment seems unnecessary to me.

Reply:
Removed in 889cab4e3d8cee5fdb584cb9e0c9d6810e714d05